pkg/krt/krt.go

```diff
@@ -124,3 +125,18 @@ func (np NetworkingProtocol) IsValid() bool {
 
 	return ok
 }
+
+type ProcessCPU struct {
+	Request string `yaml:"request"`
+	Limit   string `yaml:"limit"`
+}
+
+type ProcessMemory struct {
+	Request string `yaml:"request"`
+	Limit   string `yaml:"limit"`
+}
+
+type ProcessResourceLimits struct {
+	CPU    *ProcessCPU    `yaml:"CPU"`
+	Memory *ProcessMemory `yaml:"memory"`
+}
```

Comment:
These two types could be replaced by a generic ResourceLimits struct
```suggestion
type ResourceLimit struct {
	Request string `yaml:"request"`
	Limit   string `yaml:"limit"`
}

type ProcessResourceLimits struct {
	CPU    *ResourceLimit    `yaml:"CPU"`
	Memory *ResourceLimit `yaml:"memory"`
}
```

Reply:
Good catch